docs/implement-a-customized-actuator-en.md

```diff
@@ -0,0 +1,322 @@
+      SumContract sumContract = any.unpack(SumContract.class);
+      long param1 = sumContract.getParam1();
+      long param2 = sumContract.getParam2();
+      long sum = param1 + param2;
+
+      logger.info(String.format("\n\n" +
+                      "-------------------------------------------------\n" +
+                      "|\n" +
+                      "| SumActuator: param1 = %d, param2 = %d, sum = %d\n" +
+                      "|\n" +
+                      "-------------------------------------------------\n\n",
+              param1, param2, sum));
+      ret.setStatus(fee, code.SUCESS);
+    } catch (ArithmeticException | InvalidProtocolBufferException e) {
+      logger.debug(e.getMessage(), e);
+      ret.setStatus(fee, code.FAILED);
+      throw new ContractExeException(e.getMessage());
+    }
+    return true;
+  }
+
+  /**
+   * define the rule to validate the contract
+   *
+   * this demo first checks whether contract is null, then checks whether ${any} is a instanceof SumContract,
+   * then validates the ownerAddress, finally checks params are not less than 0.
+   */
+  @Override
+  public boolean validate() throws ContractValidateException {
+    if (this.any == null) {
+      throw new ContractValidateException("No contract!");
+    }
+    final SumContract sumContract;
+    try {
+      sumContract = any.unpack(SumContract.class);
+    } catch (InvalidProtocolBufferException e) {
+      logger.debug(e.getMessage(), e);
+      throw new ContractValidateException(e.getMessage());
+    }
+    byte[] ownerAddress = sumContract.getOwnerAddress().toByteArray();
+    if (!DecodeUtil.addressValid(ownerAddress)) {
+      throw new ContractValidateException("Invalid ownerAddress!");
+    }
+    long param1 = sumContract.getParam1();
+    long param2 = sumContract.getParam2();
+    if(param1 < 0 || param2 < 0){
+      logger.debug("negative number is not supported");
+      return false;
+    }
+    return true;
+  }
+
+  /**
+   * this method returns the ownerAddress
+   * @return
+   * @throws InvalidProtocolBufferException
+   */
+  @Override
+  public ByteString getOwnerAddress() throws InvalidProtocolBufferException {
+    return any.unpack(SumContract.class).getOwnerAddress();
+  }
+
+  /**
+   * burning fee for a contract can reduce attacks like DDoS.
+   * choose the best strategy according to the business logic
+   *
+   * here return a contant just for demo
+   * @return
+   */
+  @Override
+  public long calcFee() {
+    return TRANSFER_FEE;
+  }
+}
+```
+
+For simplicity, printing the output of SumActuator directly to log file. If there are informations that need to be stored, considering creating a new chainbase to store the data (guidance on how to create a chainbase will be released soon).
+
+After `SumActuator`, `invokeSum(MathContract.SumContract req, StreamObserver<Transaction> responseObserver)` function in RpcApiService's sub-class `WalletApi` need to be implemented to receive and process `SumContract`
```

Comment:
As xxx finished, ....

here, use `finished` to separate 2 misleading connected items. 

trailing period.